src/review/routes/build/BuildSummary.js

```diff
@@ -36,12 +34,12 @@ export function BuildSummary(props) {
   )
 }
 
-BuildSummary.propTypes = {
+BuildSummaryView.propTypes = {
   classes: PropTypes.object.isRequired,
   fetch: PropTypes.object.isRequired,
 }
 
 export default recompact.compose(
   withStyles(styleSheet),
   connect(state => state.ui.build),
-)(BuildSummary)
+)(BuildSummaryView)
```

Comment:
Can we rename the file and stylesheet name? I would rather have a strong mapping between component names and files.